node/node.go

```diff
@@ -679,6 +754,27 @@ func (a *NodeAPI) Run(ctx context.Context) error {
 		}
 	}()
 
+	// Find more peers for coordinator network here
+	if a.coordinatorNetwork && a.coordinatorNetworkFindMorePeersInterval > 0 {
+		go func() {
+			// Do an initial discovery on start up
+			if err := a.api.FindMorePeersForCoordinatorsNetwork(); err != nil {
+				log.Info("API.FindMorePeersForCoordinatorsNetwork", "err", err)
```

Comment:
```suggestion
				log.Info("API.FindMorePeersForCoordinatorsNetwork. ", "err", err)
```

Reply:
logs below this one use the same pattern (without adding `. ` at the end). Should I also change those?